configs/mirrors.txt

```diff
@@ -1,14 +1,11 @@
 url='http://resources.ovirt.org/pub',whitelist=True
-url='http://mirror.linux.duke.edu/ovirt/pub'
 url='http://mirrors.ibiblio.org/ovirt'
 url='http://mirror.isoc.org.il/pub/ovirt'
-url='http://www.gtlib.gatech.edu/pub/oVirt/pub'
```

Comment:
this one is still reachable but outdated, perhaps the syncing is not working anymore?

Reply:
Maybe there problem is just with sync, but admins haven't been answering for a long time already, so I don't see any other options than to drop them from the mirrorslist

Reply:
ok